CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # activerecord-postgres_pub_sub
 
+## v2.2.0 (Unreleased)
```

Comment:
Thanks for adding a changelog entry @TimWei! Do you mind remove the explicit `v2.2.0` heading (`Unreleased` is totally fine), as well as the change to the `VERSION` constant in this PR? We will take care of that when we release a new version!